boring/Cargo.toml

```diff
@@ -16,12 +16,15 @@ bitflags = "1.0"
 foreign-types = "0.5"
 lazy_static = "1"
 libc = "0.2"
-boring-sys = { version = ">=1.1.0,<3.0.0", path = "../boring-sys" }
+boring-sys = { version = ">=1.1.0,<3.0.0", path = "../boring-sys", default-features = false }
```

Comment:
Yeah, features are supposed to be additive, so that if one client requests "foo" and another requests "bar" the crate gets built with both. The default-features thing isn't perfect either, because "demoting" something from implicitly included to a default feature can only be done once. After that, someone else might be using `default-features = false` to mean "everything but 'foo'" but still expect "bar" to be available. I remember seeing a Cargo discussion thread about improving this, but I can't find it offhand.